src/test/kotlin/no/nav/familie/ef/sak/behandlingsflyt/steg/BeregnYtelseStegTest.kt

```diff
@@ -85,8 +88,6 @@ internal class BeregnYtelseStegTest {
             val nyAndelFom = LocalDate.of(2022, 1, 1)
             val nyAndelTom = LocalDate.of(2022, 1, 31)
 
-            val slot = slot<TilkjentYtelse>()
-            every { tilkjentYtelseService.opprettTilkjentYtelse(capture(slot)) } answers { firstArg() }
             every { tilkjentYtelseService.hentForBehandling(any()) } returns
                     lagTilkjentYtelse(listOf(lagAndelTilkjentYtelse(100, forrigeAndelFom, forrigeAndelTom)))
             every { beregningService.beregnYtelse(any(), any()) } returns listOf(lagBeløpsperiode(nyAndelFom, nyAndelTom))
```

Comment:
Trenger man å mocke BeregningService? Eller burde vi bare bruke `BeregningService()` ? 
Noen tester feiler hvis man bruker BeregniningService direkte, då feil antall perioder returneres 👀

Reply:
Helt i starten tror jeg det var annotert med `@Service`, men det gikk vi bort fra siden det ikke er noen avhengigheter. Vi har fjernet mock-bruken et par andre steder og kan sikkert gjøre det her også, men da må kanskje testene renskrives?

Reply:
Ja 😁 det burde nog gjøres i en egen PR, er litt usikker på noen av de
Lurer nesten på om AndelHistorikkBeregnerTest også hadde kunnet kjørt igjennom løypa for BeregnYtelseSteg, for å være sikker på att man faktiskt tester med riktig state av vedtak/andeler, og ikke tester en kombinasjon av vedtak og andeler som nå